spec/features/families/admin_views_all_not_families_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "Admin can see all the not active families when in families#index" do
+  context "when click on the 'Not Active' button" do
+    scenario "successfully" do
+      create(:family, active: false, last_name: "Scara")
+
+      visit families_path
+      find("#not_active_families_button").click
+
+      expect(page).to have_text(I18n.t("families.index.not_active_families"))
+      expect(page).to have_text("false")
```

Comment:
Again, I would change this to check that the active family is not displayed on the page

Reply:
Modified in sha `b6c4d17`